### ISSUE TITLE
8.11.6. New - Screen Name/Title: Settings – Add Reply-to Address - Missing Form Validation

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-planning#2478]
Component: Add Reply-to Address
Observation: 
The error message Enter an email address is visually available and referenced via aria-describedby, but there’s no assertive ARIA live region to notify screen reader users when validation fails.

(Attachment: 8.11.6. Missing Form Validation Notification for VO.mp4)

Recommendation: 
Add aria-live="assertive" to the error container to ensure screen reader users are notified of the error immediately.

**Comment by amazingphilippe** [comment]
As with other similar issues, this is not a live region, it is probably loaded after the form is submitted with errors.
